src/lib.rs

```diff
@@ -68,9 +68,17 @@ pub fn fix(args: &clap::ArgMatches, current_dir: &Path) -> Result<(), Box<dyn Er
 
         let mut lines = get_line_entries(&fe, strings);
         let mut result = checks::run(&lines, &skip_checks);
-
-        // run fixers & write results to file
-        if !result.is_empty() && fixes::run(&mut result, &mut lines, &skip_checks) > 0 {
+        let expected_fixes = result.len();
+        let mut fixes_done = 0;
+        if !result.is_empty() {
+            fixes_done = fixes::run(&mut result, &mut lines, &skip_checks);
+        }
+        // Compare amount of fixes done to amount of expected fixes. Print warning if they differ.
+        if fixes_done < expected_fixes {
+            output.print_not_all_warnings_fixes();
+        }
+        // write results to file
+        if !result.is_empty() && fixes_done > 0 {
```

Comment:
I think we can simplify this code 🤔
```suggestion
        if result.is_empty() {
            continue;
        }

        let fixes_done = fixes::run(&mut result, &mut lines, &skip_checks);
        if fixes_done != result.len() {
            output.print_not_all_warnings_fixes();
        }
        if fixes_done > 0 {
```